app/admin/page.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
[No need to import react in modern codebases](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html). Check other files as well.

Reply:
Fix the issue